main.tf

```diff
@@ -171,8 +171,36 @@ resource "aws_lb_listener" "http_forward" {
   }
 }
 
+resource "aws_lb_listener" "http_forward_ignore_default_action" {
```

Comment:
Huh this is beyond the scope of your PR, but noting for myself and any fellow contributors: I'm not even sure why we need to have two of these resources for http_forward + http_redirect. It seems we could just do the conditional logic In the default action itself without requiring two resources. 

@joe-niland do you happen to know the history behind why it was done this way?

Reply:
@Gowiem sorry I missed this one! I don't know why but it dates back to the initial implementation I think. I agree it could be refactored as you described.

Reply:
@joe-niland I'll create an issue and we can see if a passer by wants to pick it up!